### PR TITLE
Various enhancements in gallery for denoising

### PR DESCRIPTION
**Pull request by sciunto**
Follows #2240

After I merged, I realized that the wavelet looks awkward, and definitely not like the results presented by @stsievert 

toto

My changelog:
* remove double import
* fix the spacing between images
* switch to chelsea, a great picture to see the effect on thin details.

### CODE REVIEW

**Comment by stsievert** [comment]
Whoops -- I must have made a mistake. I'll give this another look tonight.

**Comment by sciunto** [comment]
thanks @stsievert It's time for me to go to bed ;)

File: doc/examples/filters/plot_denoise.py [hunk 1]
```diff
@@ -3,13 +3,13 @@
 Denoising a picture
 ====================
 
-In this example, we denoise a noisy version of the picture of the astronaut
-Eileen Collins using the total variation and bilateral denoising filter.
+In this example, we denoise a noisy version of a picture using the total
+variation, bilateral denoising and wavelet filters.
```
Comment (soupault) [inline doc/examples/filters/plot_denoise.py]: -> "total variation, bilateral, and wavelet denoising filters." ?

**Comment by stsievert** [comment]
Turns out `denoise_wavelet` needs the argument `multichannel=True`.

`denoise_wavelet` defaults to `multichannel=False`, which looks like a bad default. Should we change this? `denoise_wavelet` hasn't been included in any major release (implemented in #2190).

**Comment by soupault** [comment]
True, it is being interpreted as a 3D grayscale image.
@sciunto could you add this parameter (`multichannel=True`) to the function call, please?

**Comment by sciunto** [comment]
I tried before I left my home but I got another problem.

```
/home/fr/github/scikit-image/doc/source/../examples/filters/plot_denoise.py failed to execute correctly:Traceback (mo
st recent call last):
  File "/home/fr/github/scikit-image/doc/source/../ext/sphinx_gallery/gen_rst.py", line 455, in execute_code_block
    exec(code_block, example_globals)
  File "<string>", line 35, in <module>
  File "/home/fr/github/scikit-image/skimage/restoration/_denoise.py", line 574, in denoise_wavelet
    wavelet_levels=wavelet_levels)
ValueError: could not broadcast input array from shape (300,452) into shape (300,451)
________________________________________________________________________________

```

**Comment by sciunto** [comment]
@stsievert What's your version of pywt?

**Comment by grlee77** [comment]
@sciunto 
If you truncate the odd-sized dimension from 451 to 450 this issue will not occur.  This is a known issue with odd-sized inputs with `dwtn/idwtn` (https://github.com/PyWavelets/pywt/issues/80)

The proper fix is probably for to have `denoise_wavelet` itself either:
1.)  raise an error if any non-channels dimension of the input has odd size
or
2.)  use `np.pad` to extend to even size, call wavelet denoising, and then remove the padding

**Comment by grlee77** [comment]
actually, padding is not necessary, just truncation.  see a proposed solution in #2462

**Comment by sciunto** [comment]
Thanks @grlee77 for the explanation!

**Comment by sciunto** [comment]
@soupault I opened another ticket to fix the size issue, so you can review this one ;) (We can parallelize processes :) )

**Comment by stsievert** [comment]
I've thought about the API change proposed in https://github.com/scikit-image/scikit-image/pull/2461#issuecomment-273669621, and I'm more inclined to go with no change and defaulting to `multichannel=False`, which will work well for grayscale images. `multichannel=True` will work well for color images.

@JDWarner, @soupault and I did discuss this at https://github.com/scikit-image/scikit-image/pull/1833#issuecomment-167035325 and arrived at the same conclusion.

**Comment by sciunto** [comment]
@stsievert Thanks for the clarification. Then, it would be good to deprecate the default value of denoise_bilateral to be consistent. I open a ticket.

**Comment by soupault** [comment]
Thanks Francois!

**Pull request merged**